lib/model/Tokens.js

```diff
@@ -95,6 +95,34 @@ Tokens.prototype.connectByApiToken = function(tenantName, apiToken, callback) {
     });
 }
 
+/**
+ * Authenticate with the API endpoint to get the entire token object
+ * 
```

Comment:
@cratter49 This isn't actually authenticating, from what I can see. Can you please correct the JS Doc?

Reply:
This appears to be the "fetch token details" api endpoint that is conveniently omitted from the docs here: http://docs.soasta.com/repository-api/#tokens

Might be a good idea to update the API docs as well.